book/src/user_manual/create_group.md

```diff
@@ -10,14 +10,10 @@ In practice, the supported and required extensions are set by adding them to the
 {{#include ../../../openmls/tests/book_code.rs:create_key_package_bundle}}
 ```
 
-Every group has a unique group ID that needs to be specified at the time of the group creation. The group ID cannot be changed after the group creation and therefore remains immutable throughout the group's lifetime. It should be chosen so that it doesn't collide with any other group IDs in the same system:
```

Comment:
Can you update this text (and/or the note below) to reflect that there are now two constructors and what they do? Ideally, you can also provide examples by using the constructors in the book code.

Reply:
Ah, yeah. That slipped through :+1: